### ISSUE TITLE
no-multiple-empty-lines should ignore template strings

### ISSUE THREAD

**Issue by davidtheclark** [eslint/eslint#2605]
Right now I'm getting warnings from this rule if I have the following:

```js
const str = (
`first


second`);
```

But because that's essentially a string, I'd think the rule should ignore that, allow whitespace to be arbitrary there as it would ignore `"first\n\n\n\nsecond"`, right?

**Comment by ilyavolodin** [comment]
Can you provide more details please? Which version of eslint? Do you by any chance use `bable-eslint`? We had similar reports for it.

**Comment by davidtheclark** [comment]
Apologies. I reduced what had going to a simple test case, and can see that I'm getting the error with ES Lint v0.21.2 without babel-eslint, with this config:

```
env:
  es6: true

rules:
  no-multiple-empty-lines: [2, {"max": 1}]
```

**Comment by xjamundx** [comment]
Yeah I can see how this would happen ;)

**Comment by lo1tuma** [comment]
A bit late, but shouldn't we make the same exception for multiline strings
and multiline comments?

**Comment by ilyavolodin** [comment]
Umm.. That's a good point. I'm not 100% sure on multi-line comments, but multi-line string I think should be covered. I'll create a new issue for this.
